controlplane/kubeadm/controllers/remediation_test.go

```diff
@@ -361,6 +399,44 @@ func TestCanSafelyRemoveEtcdMember(t *testing.T) {
 
 		g.Expect(testEnv.Cleanup(ctx, m1, m2, m3)).To(Succeed())
 	})
+	t.Run("Can safely remediate 3 machines CP when the etcd member being remediated is missing", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("Can safely remediate 3 machines CP when the etcd member being remediated is unhealthy", func(t *testing.T) {
```

?

Reply:
I don't think this should be fixed because because we are removing the member to be remediated from the list of members we pass to the fakeManagementCluster (so the member is missing)